hexrd/material.py

```diff
@@ -731,7 +731,10 @@ def dump_material(self, file, path=None):
         AtomInfo['stiffness'] = self.unitcell.stiffness
         AtomInfo['hkls'] = self.planeData.getHKLs()
         AtomInfo['dmin'] = self.unitcell.dmin
-        AtomInfo['tThWidth'] = numpy.degrees(self.planeData.tThWidth)
+        if self.planeData.tThWidth is None:
+            AtomInfo['tThWidth'] = Material.DFLT_TTH
```

Comment:
Isn't `Material.DFLT_TTH` in radians? I think you need to convert it to degrees here.

Reply:
@saransh13 -- have you verified?

Reply:
In the two lines below this, the plane data `tThWidth` is being converted to degrees. The `Material.DFLT_TTH` must be converted to degrees as well.